client/src/views/header/NotificationCenterModal.vue

```diff
@@ -0,0 +1,223 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="notification-center-container">
+    <div class="notification-center">
+      <div class="notification-center-header">
+        <ion-text class="notification-center-header__title title-h4">
+          {{ $msTranslate('notificationCenter.title') }}
+        </ion-text>
+        <span class="notification-center-header__counter body-sm">
+          {{ unreadCount > 99 ? '+99' : unreadCount }}
+        </span>
+        <div class="notification-center-header-actions">
+          <ion-toggle
+            class="notification-center-header-actions__toggle dark small body-sm"
+            @ion-change="onlyReadToggle = !onlyReadToggle"
+          >
+            {{ $msTranslate('notificationCenter.readOnly') }}
+          </ion-toggle>
+          <ion-icon
+            class="notification-center-header-actions__close"
+            :icon="close"
+            @click="closeModal()"
```

Comment:
You can just call `modalController.dismiss()` and delete the `closeModal` function since it doesn't do anything more